src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -67,6 +68,7 @@ public function singularifyProvider()
             array('emphases', array('emphas', 'emphase', 'emphasis')),
             array('faxes', 'fax'),
             array('feet', 'foot'),
+            array('feedback', 'feedback'),
```

Comment:
I don't think Symfony has to worry about it? (or maybe I'm not fully aware of the usage of this class) Does it matter to Symfony whether `getFeedback()` is a plural or a singular? It might not be the best choice because of this, but that's a problem of the project imo.